### PR TITLE
Views and typeahead for subdomain

### PR DESCRIPTION
**Pull request by sabrina-ulbrich**
This branch works on staging. Some more little changes would be nice (i.e. logo that stick out a bit from header, but that has some issues with the flex box, would be cool to repair that for speakerinnen anyway). Apart from that this is the final version for the subdomain.

### CODE REVIEW

File: app/assets/stylesheets/pages.scss [hunk 1]
```diff
@@ -34,6 +27,74 @@
...
+.vorarlberg footer {
+  padding-bottom: 3rem;
+}
+
+.vorarlberg footer figure {
+  margin: 0;
+}
+
+.vorarlberg footer .logo-1 {
+  width: 180px;
```
Comment (zaziemo) [inline app/assets/stylesheets/pages.scss:81]: The footer could be improved for smaller screen sizes

File: app/controllers/profiles_controller.rb [hunk 2]
```diff
@@ -102,7 +102,7 @@ def render_footer?
 
   def typeahead
     suggester_options = []
-    suggestions = Profile.typeahead(params[:q])['suggest']
+    suggestions = Profile.typeahead(params[:q], current_region)['suggest']
```
Comment (zaziemo) [inline app/controllers/profiles_controller.rb:105]: Let's quickly talk through this. Where does current region come from? What if it is not set?

File: app/helpers/search_helper.rb [hunk 3]
```diff
@@ -10,13 +10,15 @@ def all_states
...
+    filter_params = {
+      search: params[:search],
+      filter_countries: params[:filter_countries],
+      filter_cities: params[:filter_cities],
+      filter_lang: params[:filter_lang],
+      filter_states: params[:filter_states]
+    }
 
-    [
+    hashes = [
```
Comment (zaziemo) [inline app/helpers/search_helper.rb:21]: probably a more describing name like `aggregation_hashes`?

File: app/helpers/search_helper.rb [hunk 4]
```diff
@@ -58,5 +60,9 @@ def filters_hash(aggs_countries, aggs_states, aggs_cities, aggs_languages)
         link_text_func: ->(term_key) { t(term_key, scope: 'iso_639_1').capitalize }
       }
     ]
+
+    hashes.reject do |hash|
+      (hash[:key] == 'countries' || hash[:key] == 'states') && !current_region.nil?
```
Comment (zaziemo) [inline app/helpers/search_helper.rb:65]: instead of `!current_region.nil?` `current_region.present?`

File: app/views/layouts/_footer.html.erb [hunk 5]
```diff
@@ -1,57 +1,121 @@
...
-          </li>
-          <li class="d-flex">
-            <%= link_to t(:title, scope: 'pages.imprint'), impressum_path, class: ""%>
-          </li>
-          <li class="d-flex">
-            <%= link_to t(:privacy, scope: 'pages.imprint'), privacy_path, class: "" %>
-          </li>
-        </ul>
-      </div>
+      <% if current_region == nil %>
```
Comment (zaziemo) [inline app/views/layouts/_footer.html.erb:4]:
```suggestion
      <% if current_region.nil? %>
```

File: app/views/layouts/_footer.html.erb [hunk 6]
```diff
@@ -1,57 +1,121 @@
...
+              <a href="https://www.facebook.com/MarkeVorarlberg/", target='_blank'>Facebook: @markevorarlberg</a><br>
+              <a href="https://www.instagram.com/marke_vorarlberg/?hl=de", target='_blank'>Instagram: @marke_vorarlberg</a>
+            </figcaption>
+          </figure>
+        </div>
+        <div class="col-12 col-md-3 pl-3">
+          <figure>
+            <%= link_to(image_tag('logo_gleichstellung.png', class:'logo-2'), "https://vorarlberg.at/frauen",target: 'blank' ) %>
+            <figcaption class="pt-4">
+              vorarlberg.speakerinnen.org wird unterstützt vom
```
Comment (zaziemo) [inline app/views/layouts/_footer.html.erb:71]: shouldn't this also be translated into english?

File: app/views/layouts/_footer.html.erb [hunk 7]
```diff
@@ -1,57 +1,121 @@
...
+                <%= link_to t(:imprint, scope: 'pages.footer'), impressum_path %>
+              </li>
+              <li class="d-flex">
+                <%= link_to t(:privacy, scope: 'pages.footer'), privacy_path %>
+              </li>
+            </ul>
+          </div>
+          <div class="credits col-12 col-md-12">
+            <p>
+              Bildachweis: Architektin Adelheid Gnaiger, Fotografie: Walter Gnaiger, siehe auch:
```
Comment (zaziemo) [inline app/views/layouts/_footer.html.erb:112]: translation?

File: app/views/shared/_categories.html.erb [hunk 8]
```diff
@@ -1,16 +1,25 @@
...
-    <span class="child">
-      <div class="h4 category_name text-left pl-5 sans-serif"><%= category.name %></div>
-    </span>
-    <div class="progress-bar category_item_<%=category.short_name%>" role="progressbar" style="width: <%=bar_1%>%" aria-valuenow="<%=bar_1%>" aria-valuemin="0" aria-valuemax="100">
-    </div>
-    <div class="category_item_<%=category.short_name%>_numbers pr-5" role="progressbar" style="width: <%=bar_2%>%" aria-valuenow="<%=bar_2%>" aria-valuemin="0" aria-valuemax="100">
-      <div class="h4 category_count text-right pr-5 pt-3"><%= category_profiles_count(category.id).to_s + " " + t(:speakerinnen, scope: 'pages.home') %></div>
-    </div>
-  </a>
+<% if current_region == nil %>
```
Comment (zaziemo) [inline app/views/shared/_categories.html.erb:1]:
```suggestion
<% if current_region.nil? %>
```

File: app/views/shared/_search_filter.erb [hunk 9]
```diff
@@ -46,18 +46,26 @@
...
-      <br><br>
-      <ul class="list-unstyled">
-        <%= t(:random_categories, scope: 'search' ).html_safe %><br>
-        <% @three_sample_categories.each do |cat| %>
-          <li>
-            <%= category_link(cat, "speakers_anchor") %>
-          </li>
-        <% end %>
-      </ul>
+      <% if current_region == nil %>
```
Comment (zaziemo) [inline app/views/shared/_search_filter.erb:49]:
```suggestion
      <% if current_region.nil? %>
```

File: app/views/shared/_sub_menu.erb [hunk 10]
```diff
@@ -4,38 +4,68 @@
...
 </div>
 <div class="col-md-6 col-lg-3 mb-2">
   <div class="card pb-3 bg--white-trans">
-    <div class="card-header">
-      <h3 class="card-title mb-0"><%= link_to t(:sub_menu_blog_title, scope: 'pages.home'), "https://blog.speakerinnen.org/", target: '_blank' %></h3>
-    </div>
-    <div class="card-body">
-      <p class="h5 card-text mt-2"><%= t(:sub_menu_blog_text, scope: 'pages.home').html_safe %></p>
-    </div>
+    <% if current_region == nil %>
```
Comment (zaziemo) [inline app/views/shared/_sub_menu.erb:55]:
```suggestion
    <% if current_region.nil? %>
```

File: config/locales/pages.de.yml [hunk 11]
```diff
@@ -32,73 +36,39 @@ de:
...
+      welcome_title_vab: "Herzlich willkommen."
+      welcome_text: "Ein herzliches Willkommen! Hier findest du unsere Neuzugänge"
+      welcome_text_vab: "Unsere Neuzugänge aus Vorarlberg:"
+      intro_title_vab: "Wir machen Expertinnen sichtbar."
+      intro_text_vab: |
+        <p>
+          vorarlberg.speakerinnen.org ist ein Projekt zur Verhinderung von 'manels',
+          also Podien, Gremien oder Panels, die vornehmlich mit Männern besetzt sind.
+          Durch die Kooperation mit <a href="https://speakerinnen.org", target="_blank">speakerinnen.org</a>
+          werden Vorarlberger Frauen* sowohl lokal wie international als Expertinnen, Rednerinnen und
```
Comment (zaziemo) [inline config/locales/pages.de.yml:52]: We decided to remove the * behind Frauen.

**Review by zaziemo — COMMENTED** [review]
Let's talk through these changes quickly. I have some small questions.

**Comment by sabrina-ulbrich** [comment]
@zaziemo I added the changes we talked about. After deployment, `bundle exec rake elasticsearch:import:all` is necessary.

**Review by zaziemo — APPROVED** [review]
Good work :)

**Pull request merged**